src/commands/env/api.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { readFile } from 'node:fs/promises';
+import got, { Headers, Method } from 'got';
+import * as chalk from 'chalk';
+import * as ProxyAgent from 'proxy-agent';
+import { getProxyForUrl } from 'proxy-from-env';
+import { SfCommand, Flags } from '@salesforce/sf-plugins-core';
+import { SfError, Messages, Org } from '@salesforce/core';
+import { CliUx } from '@oclif/core';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.load('@cristiand391/sf-plugin-api', 'env.api', [
+  'summary',
+  'description',
+  'examples',
+  'flags.target-org.summary',
+  'flags.method.summary',
+  'flags.include.summary',
+  'flags.header.summary',
+  'flags.body.summary',
+  'errors.invalid-http-header',
+]);
+
+export default class EnvApi extends SfCommand<string> {
+  public static summary = messages.getMessage('summary');
+  public static description = messages.getMessage('description');
+  public static examples = messages.getMessages('examples');
+  public static enableJsonFlag = false;
+  public static flags = {
+    'target-org': Flags.requiredOrg({
+      summary: messages.getMessage('flags.target-org.summary'),
+      char: 'o',
+    }),
+    include: Flags.boolean({
```

Comment:
`include` could have the short character `i` to match `curl`

Reply:
Added ✅ 

curl also has `-I/--head` which makes it only prints the headers and skip the response, I'll add a flag for that later, sometimes you are interested on the headers and if the response is big you need to scroll to the top.